packages/chrome/src/views/subnav/CollapsibleSubNavItem.js

```diff
@@ -14,6 +14,7 @@ import { retrieveTheme } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'chrome.collapsable_sub_nav_item';
 const PANEL_COMPONENT_ID = 'chrome.collapsable_sub_nav_item_panel';
+const SUB_NAV_ITEM_HEIGHT_PX = 38;
```

Comment:
Item height is variable – can wrap to multiple lines. I was hoping this measurement could be obtained and applied post-render.

Reply:
If we are animating the `max-height` over time, how would we calculate that total height without having a render flash? Lets discuss offline.

Reply:
On post-render, iterate over the children and get `element.offsetHeight + top-margin + bottom-margin` for each. Apply the value to the panel as `style="max-height: [value]"`.

Reply:
I just pushed an update that adds a new `AccordionContainer` component. This abstracts all of the accessibility logic required for the [W3C accordion pattern](https://www.w3.org/TR/wai-aria-practices/#accordion)

I also calculated the panel `maxHeight` using a slightly different approach. Lets discuss offline.